terraform.tfvars.example

```diff
@@ -60,6 +60,9 @@ licenseServerIaaSAntimalware = true
 # For using this extension logAnalyticsWorkspaceName and logAnalyticsWorkspaceResourceGroupName needs to be set.
 licenseServerMicrosoftMonitoringAgent = true
 
+# Enables instalation of Microsoft Guest configuration extension on license server VM.
```

Comment:
As in variables.tf: I'd find it helpful for an admin to mention in the comment that the following variables depend on the licenseServer variable:
- licenseServerIaaSAntimalware
- licenseServerMicrosoftMonitoringAgent
- licenseServerMicrosoftGuestConfiguration